apps/src/panels/panelsView.module.scss

```diff
@@ -108,6 +108,16 @@ $text-horizontal-padding: 2.6cqw;
         }
       }
 
+      ol, ul {
+        font-size: 1.6cqw;
```

Comment:
Would it make sense to extract `1.6cqw` as a constant so it's consistent with the paragraph and text font size?

Reply:
Good call.  Might save that for a bigger refactoring.